docs/reference/analysis/tokenfilters/synonym-graph-tokenfilter.asciidoc

```diff
@@ -191,16 +191,16 @@ If the stop filter removed `baz` instead:
 - If `lenient` is set to `false`, an error will be raised as `baz` would be removed from the right hand side of the synonym rule.
 - If `lenient` is set to `true`, the mapping will have no effect as the target word is removed.
 
-For *equivalent mappings* like `foo, bar, baz` with a stop filter that removes `bar`:
+For *equivalent synonyms* like `foo, bar, baz` with a stop filter that removes `bar`:
 
 - If `lenient` is set to `false`, an error will be raised as `bar` would be removed from the synonym rule.
-- If `lenient` is set to `true`, the mappings added would be equivalent to `foo => baz` and `baz => foo`, which do not contain the removed word.
+- If `lenient` is set to `true`, the mappings added would be equivalent to `foo => foo` and `baz => foo`, which do not contain the removed word.
```

Comment:
This updated language misses expanding `foo` to `baz` and and that the `baz` token is retained. Maybe we use this list of explicit synonyms to be complete?

```
foo => foo
foo => baz
baz => foo
baz => baz
```

Reply:
Makes sense, thanks!